spec/models/item_spec.rb

```diff
@@ -12,8 +12,12 @@
   it { should_not have_valid(:description).when('', nil) }
 
   context 'uniqueness' do
-    let!(:item) { FactoryGirl.create(:item) }
-    let!(:dupe_item) { FactoryGirl.create(:item) }
+    let!(:user) { FactoryGirl.create(:user) }
+    let!(:category) { FactoryGirl.create(:category) }
+    let!(:item) { FactoryGirl.create(:item, user: user, category: category) }
+    let!(:dupe_item) { FactoryGirl.create(:item,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.